docs/open-source-tizen/porting/application.md

```diff
@@ -1,12 +1,14 @@
 # Application
 
-Tizen supports both core and reference applications. The core applications are developed with platform internal interfaces, such as Enlightenment Foundation Libraries EFL and other 3rd party libraries. The reference applications are developed with Tizen native APIs.
+Tizen supports both core and reference applications. Core applications are developed with platform internal interfaces, such as the Enlightenment Foundation Libraries (EFL) and other third party libraries. Reference applications are developed with Tizen native APIs.
 
-The following table shows whether the core and reference versions of the preloaded sample applications are supported by default on the Emulator and target device.
+The following table shows whether the core and reference versions of the preloaded sample applications are supported by default on the emulator and target device.
 
-| Application name | Emulator              | Target           |                       |      |
+**Table: Sample application support**
```

Comment:
We have followed the usual Tizen documentation writing rules.
And according to those, all tables need a caption.
Also, all images need a caption, except for screenshots used in step lists.
There are a few known exceptions for these rules, but 99% of the current tables and images follow it.

You seem to be now suggesting that the writing rules need to be changed.
If so, do you want to change the rules just for the open source content, or also for the basic guide/training/studio content that is currently handled in Gerrit?
And if the rules are changed, what exactly would be the new rule? That captions are used in tables and figures only when absolutely necessary for understanding them?

This change is not a simple one. In the existing documentation, an introduction sentence is often omitted before a table or figure, simply because the caption explains what the table or figure is. If the new rule is to avoid captions, does this mean that when something is needed to clarify the content, an intro sentence is a better option than a caption? Or is it then still better to have a caption than an intro sentence, if one is definitely needed?

Reply:
I understand you and know the old writing rules of Tizen documents guides. 
It is okay you provide a caption.

Most Tizen developer will write and update docs directly on GitHub. The purpose of changing documents management system is to write easy and more docs anytime.
So..I hope we don't force strict format of docs.

For tutorials (Not guides) that explains how to use step by step, we don't need to provide a caption for a screenshot.
For example: 
https://developer.tizen.org/development/tizen-.net-preview/getting-started/installing-visual-studio-tools-tizen

Others, we need to provide a caption. 

Conclusion, I think a caption is not a mandatory. A unnecessary caption makes us hard to update.

Reply:
Please confirm whether we understand this correctly:

In the future, a caption is still always added, except for screenshots and tables within a step list.

So we will change the existing rule so that any table captions within step lists need to be removed. We will implement this rule change within the files in this push request as soon as possible.
However, we assume that this change is not a high priority, and implementing it elsewhere in the Tizen documentation can wait until more urgent tasks are done. 

Is that OK?